JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsScreen.kt

```diff
@@ -479,34 +480,65 @@ private fun InterestsTabRow(
     updateSection: (Sections) -> Unit,
     tabContent: List<TabContent>
 ) {
-    // TODO: Make the tabs narrower and aligned to the left in large screens
-    TabRow(
-        selectedTabIndex = selectedTabIndex,
-        backgroundColor = MaterialTheme.colors.onPrimary,
-        contentColor = MaterialTheme.colors.primary
-    ) {
-        tabContent.forEachIndexed { index, tabContent ->
-            val colorText = if (selectedTabIndex == index) {
-                MaterialTheme.colors.primary
-            } else {
-                MaterialTheme.colors.onSurface.copy(alpha = 0.8f)
+    BoxWithConstraints {
+        val windowSize = remember(maxWidth) { getWindowSize(maxWidth) }
+        when (windowSize) {
+            WindowSize.Compact -> {
+                TabRow(
+                    selectedTabIndex = selectedTabIndex,
+                    backgroundColor = MaterialTheme.colors.onPrimary,
+                    contentColor = MaterialTheme.colors.primary
+                ) {
+                    InterestsTabRowContent(selectedTabIndex, updateSection, tabContent)
+                }
             }
-            Tab(
-                selected = selectedTabIndex == index,
-                onClick = { updateSection(tabContent.section) },
-                modifier = Modifier.heightIn(min = 48.dp)
-            ) {
-                Text(
-                    text = stringResource(id = tabContent.section.titleResId),
-                    color = colorText,
-                    style = MaterialTheme.typography.subtitle2,
-                    modifier = Modifier.paddingFromBaseline(top = 20.dp)
-                )
+            else -> {
```

Comment:
Minor nit: I personally prefer to always add in all cases directly, rather than use an `else`. Once the exhaustive change lands in Kotlin, that will cause a compiler exception initially if a new size class is added or one is removed, which allows re-evaluating all of the decisions like this without having to search for them.

Reply:
Done, thanks!